### PR TITLE
Touch up email text

### PR DESCRIPTION
**Pull request by jsf9k**

### CODE REVIEW

File: body.html [hunk 1]
```diff
@@ -4,10 +4,20 @@
...
-    <p>Thank you</p>
+    <p>
+      Thank you,<br />
+      The NCATS team
+    </p>
+
+    <p>
+      National Cybersecurity Assessments and Technical Services (NCATS)<br />
+      National Cybersecurity and Communications Integration Center<br />
+      U.S. Department of Homeland Security<br />
```
Comment (dav3r) [inline body.html]: Shouldn't we change our signature block to something without NCCIC and DHS, that mentions CISA?  NCATS can stay for now I think.

**Review by dav3r — APPROVED** [review]
Approved, but with a suggestion

**Comment by bjb28** [comment]
@jsf9k, I think this looks good and should meet FNRs needs.

File: body.html (hunk 1 above)
Comment (jsf9k) [reply body.html]: Good point, @dav3r.  How do you like the signature block after commit 993d291?

I will update cisagov/cyhy-mailer as well.

Comment (dav3r) [reply body.html]: That looks great to me!  Good call on updating cyhy-mailer similarly.

**Pull request merged**